Simplified/NYPLBookAcquisitionPath.h

```diff
@@ -12,6 +12,7 @@ static NSString * const _Nonnull ContentTypeFindaway = @"application/vnd.library
 static NSString * const _Nonnull ContentTypeOpenAccessAudiobook = @"application/audiobook+json";
 static NSString * const _Nonnull ContentTypeOpenAccessPDF = @"application/pdf";
 static NSString * const _Nonnull ContentTypeFeedbooksAudiobook = @"application/audiobook+json;profile=\"http://www.feedbooks.com/audiobooks/access-restriction\"";
+static NSString * const _Nonnull ContentTypeOctetStream = @"application/octet-stream";
```

Comment:
can you move all these definitions to the `.m`?  It's wrong to have static variables in a header file.

Reply:
sure, done